admin/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+</head>
+<body>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
+  <link href="/admin/config.yml" type="text/yaml" rel="cms-config-url">
```

Comment:
@eddietejeda do you know the reason behind adding this line?

Reply:
Which line? config? I just copied the way it was done in the documentation. 

This will probably change to loading from local URL.

Reply:
the `link` I don't see it in the docs, maybe its necessary if the config is not in the default place, lets remove it.

Reply:
https://www.netlifycms.org/docs/configuration-options/